docs/community/com-troubleshooting/com-issue-sql.md

```diff
@@ -1,17 +1,21 @@
 ---
-title: Issue running SQL
+title: Issue running SQL over HTTPS
 layout: default
-parent: Troubleshooting
+parent: Community troubleshooting
 grand_parent: Community
-nav_order: 2
 ---
 
-# Issue - SQL won't on FeatureBase Community
+# Issue - Running SQL over HTTPS fails
 
-Statements using SQL found in [SQL-guide](/docs/sql-guide/sql-guide-home) do not execute when run:
-* from the FeatureBase Community command-line
-* via HTTP requests
+## Cause
+
+FeatureBase CLI is not enabled.
```

Comment:
Note this is enabled by default now, so it might be worth mentioning

Reply:
I've rejigged to give a version number so hopefully that makes it clearer :)